.changeset/config.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": ["@svitejs/changesets-changelog-github-compact", { "repo": "sveltejs/svelte" }],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "version-4",
+  "bumpVersionsWithWorkspaceProtocolOnly": true,
+  "ignore": []
```

```suggestion
	"ignore": ["!(@sveltejs/*|svelte)"]
```

negated ignore pattern like in sveltekit to prevent unrelated packages from showing up in cli